### PR TITLE
support for vJunosEvolved

### PR DESCRIPTION
**Pull request by akielaries**
Adds support for vJunosEvolved.  I used the below topology file with containerlab for testing. Once this is verified and upstream for vrnetlab I will make the PR for containerlab as well.

```yml
name: srlvjunos02

topology:
  nodes:
    srl:
      kind: nokia_srlinux
      image: ghcr.io/nokia/srlinux:23.7.1
      startup-config: srl.cli

    vevo:
      kind: vr-juniper_vjunosevolved
      image: vrnetlab/vr-vjunosevolved:23.2R1-S1.8-EVO
      startup-config: vjunos.cfg

  links:
    - endpoints: ["srl:e1-1", "vevo:eth1"]
    - endpoints: ["srl:e1-2", "vevo:eth2"]
    - endpoints: ["srl:e1-3", "vevo:eth3"]
```

### CODE REVIEW

**Comment by hellt** [comment]
Thanks @akielaries 
Do you mind posting here the clab binary you used for testing so that more people could give it a try?

**Comment by akielaries** [comment]
@hellt I had built my containerlab binary from this branch: https://github.com/akielaries/containerlab/tree/vJunosEvolved

File: vjunosevolved/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# vrnetlab / Juniper vJunosEvolved
+
+This is the vrnetlab docker image for Juniper's vJunosEvolved.
+
+> Available with [containerlab](https://containerlab.dev) as vr-vjunosevolved.
```
Comment (hellt) [inline vjunosevolved/README.md:5]:
```suggestion
> Available with [containerlab](https://containerlab.dev) as juniper_vjunosevolved.
```

We are drifting away from using `vr-*` kind names in favor of `vendor_nos` type of naming.

File: vjunosevolved/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:20.04
```
Comment (hellt) [inline vjunosevolved/docker/Dockerfile:1]: let's test it with `debian:bookworm-slim` as this is our current base image.

**Comment by hellt** [comment]
thnx @akielaries

**Pull request merged**